Kernel/Memory/MemoryManager.cpp

```diff
@@ -416,7 +416,7 @@ UNMAP_AFTER_INIT void MemoryManager::initialize_physical_pages()
 {
     m_global_data.with([&](auto& global_data) {
```

Comment:
The commit title should be "Kernel/Memory: Map framebuffer and address space <4GiB"

Reply:
Done